.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: cargo build --verbose
+      - name: Run tests
+        run: cargo test --verbose
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install latest rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          default: true
+          override: true
+
+      - name: Build
+        run: cargo build --all --release && strip target/release/rawsemble && mv target/release/rawsemble target/release/rawsemble_amd64
+
+      - name: Release
```

Comment:
I don't think we want to create a new Github release for every push. Instead, let's store the build result as an Artifact? https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts

Reply:
Btw the release is only run if you are pushing a tag.  

```
if: startsWith(github.ref, 'refs/tags/')
```